### ISSUE TITLE
Building the project to add functionality

### ISSUE THREAD

**Issue by antony** [krystalcampioni/vue-hotel-datepicker#87]
### Description

I'm trying to work on this project, to attempt to add SSR support. However, I can't seem to get the basics working.

### Code sample 

I've checked out the module, and done an `npm link` on it.
I made my changes.
I've run npm run build to get the latest version into /dist

But now I get this error:

```bash

 warning  in ./components/Search.vue

90:21-36 "export 'default' (imported as 'HotelDatePicker') was not found in 'vue-hotel-datepicker/dist/vue-hotel-datepicker.min.js'
```
### Datepicker Version

github/master

**Comment by amulchinock** [comment]
@krystalcampioni Should this be closed now?

**Comment by krystalcampioni** [comment]
@amulchinock I guess @antony is trying to use npm link in order to use the datepicker in another project that uses SSR, in order to work on it as a separate package in dev mode.
Not sure why it's not working though, I'll try to replicate this error.

**Comment by antony** [comment]
@krystalcampioni yeah - that's essentially it. I wanted to try out my usecase as I developed the project.

**Comment by amulchinock** [comment]
@antony @krystalcampioni Intriguing. I'm slightly embarrassed to admit that I've not come across SSR before now. Any reading material someone could point me at?

**Comment by krystalcampioni** [comment]
@amulchinock don't be embarrassed! With the speed new things are created everyday in the tech field ( and specially in the frontend field ) it's hardly impossible to come across everything. If you'd like to learn more about SSR with Vue, a great start is Nuxt.js:
https://nuxtjs.org/

If you'd rather watch a course instead of reading docs, I can surely recommend Sarah Drasner's intro to Vuejs course at Frontend Masters, which also covers SSR rendering with Nuxt:
https://frontendmasters.com/courses/vue/
